src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx

```diff
@@ -234,14 +202,116 @@ These Support Plans apply only to your paid service subscription under an existi
           </td>
         </tr>
       </tbody>
-    </table>
+</table>
+
+### Response times 
+
+The table below shows estimated response times based on your New Relic usage plan. 
```

Comment:
@homelessbirds I think Standard, Pro, and Enterprise are called "Editions"--not usage plans. See this doc: https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/

Reply:
Fixed this!